change/@ni-nimble-components-788ed1e7-ebcc-469f-ab96-b5e608276b80.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Add way to skip specific tests on Firefox",
```

Comment:
There are now source changes that should be documented and have a version type

Reply:
Yep, changed.